akka-http-core/src/main/scala/akka/http/scaladsl/settings/ConnectionPoolSettings.scala

```diff
@@ -76,6 +76,35 @@ abstract class ConnectionPoolSettings extends js.ConnectionPoolSettings { self:
 }
 
 object ConnectionPoolSettings extends SettingsCompanion[ConnectionPoolSettings] {
+
+  /**
+   * We want to support globbing on hostnames, but I didn't want to write an entire globbing engine. This assumes that
+   * *.host.name will match host.name (without the leading period) because that was most intuitive to me. I'm not a huge
+   * fan of this, open to suggestions in this PR
+   */
+  private[akka] def hostMatches(host: String, pattern: String): Boolean = {
+    pattern.replaceFirst("^*\\.", "*.?").replace(".", raw"\.").replace("*", ".*").r.pattern.matcher(host).matches()
```

Comment:
how about a simple `host endsWith myThingy` (so no * globbing but simple suffix matching)?

Reply:
I think the globbing would be worthwhile, but I'm ok with doing an endswith to start. Are you suggesting ONLY do an endswith? Or are you saying that if  it starts with `*.`, then do an endswith?